queue_services/business-pay/devops/gcp/clouddeploy.yaml

```diff
@@ -26,7 +26,7 @@ serialPipeline:
       verify: false
    deployParameters:
    - values:
-      deploy-env: "development"
+      deploy-env: "production"
```

Comment:
NOTE: if this is set to anything else then the 'flags' does not get initialized. I could alter that logic instead if it is preferred.

This sets the app to run off the 'ProdConfig' instead of the 'DevConfig'

Reply:
think your change should be fine